libraries/protocol/include/golos/protocol/config.hpp

```diff
@@ -23,7 +23,7 @@
 
 #define STEEMIT_GENESIS_TIME                    (fc::time_point_sec(1476788400))
 #define STEEMIT_MINING_TIME                     (fc::time_point_sec(1451606400))
-#define STEEMIT_CASHOUT_WINDOW_SECONDS          (60*60) /// 1 hour
+#define STEEMIT_CASHOUT_WINDOW_SECONDS          (2*60) /// 1 hour
```

Comment:
Fix comment or number.

Reply:
@afalaleev It is for test purposes. same as a magic number above.